src/code/z_schedule.c

```diff
@@ -0,0 +1,282 @@
+#include "global.h"
+
+#define SCHEDULE_CALC_TIME(hour, minute, dest, temp) \
+    (temp) = (hour)*60.0f;                           \
```

Comment:
```suggestion
    (temp) = (hour) * 60.0f;                           \
```
Minor formatting

Reply:
Actually, clang-format forces it to be the version without spaces. I guess because it is in a macro? I figured it was too minor a thing to disable formatting for the block